<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DapGuiButtonUI</class>
 <widget class="QWidget" name="DapGuiButtonUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="vl">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="DapGuiWidget" name="DapGuiButtonBackground" native="true">
     <property name="styleSheet">
      <string notr="true">#DapGuiButtonBackground{ background-color: rgba(0,0,0,0); }</string>
     </property>
     <property name="cssStyle" stdset="0">
      <string>nobackground</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="DapGuiWidget" name="Style0" native="true">
        <property name="cssStyle" stdset="0">
         <string>nobackground</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="DapGuiLabel" name="DapGuiButtonTextMain">
           <property name="text">
            <string>Main Text</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignHCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="DapGuiLabel" name="DapGuiButtonTextSub">
           <property name="text">
            <string>Sub Text</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="DapGuiWidget" name="Style1" native="true">
        <property name="cssStyle" stdset="0">
         <string>nobackground</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="DapGuiLabel" name="DapGuiButtonTextSub_2">
           <property name="text">
            <string>Sub Test</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignHCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="DapGuiLabel" name="DapGuiButtonTextMain_2">
           <property name="text">
            <string>Main Text</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="DapGuiWidget" name="Style2" native="true">
        <property name="cssStyle" stdset="0">
         <string>nobackground</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="DapGuiLabel" name="DapGuiButtonTextLeft">
           <property name="text">
            <string>$0.00</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="DapGuiLabel" name="DapGuiButtonTextMain_3">
             <property name="text">
              <string>Main Text</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="DapGuiLabel" name="DapGuiButtonTextSub_3">
             <property name="text">
              <string>Sub Text</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="DapGuiWidget" name="Style3" native="true">
        <property name="cssStyle" stdset="0">
         <string>nobackground</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="DapGuiLabel" name="DapGuiButtonIcon">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="DapGuiLabel" name="DapGuiButtonTextMid">
           <property name="text">
            <string>Main Text</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="DapGuiLabel" name="DapGuiButtonTextRight">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Sub Text</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="DapGuiWidget" name="Style4" native="true">
        <property name="cssStyle" stdset="0">
         <string>nobackground</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="DapGuiLineEdit" name="kelGuiLineEditMain">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string>Main Text</string>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="DapGuiLabel" name="DapGuiButtonTextSub_4">
           <property name="text">
            <string>Sub Text</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="DapGuiWidget" name="Style6" native="true">
        <property name="cssStyle" stdset="0">
         <string>nobackground</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="DapGuiLabel" name="DapGuiButtonIcon_61">
           <property name="cssStyle" stdset="0">
            <string>ch-country-filter-button</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="DapGuiLineEditNative" name="kelGuiLineEditMainNative">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string>Main Text</string>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="DapGuiWidget" name="Style5" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="DapGuiLabel" name="DapGuiButtonIconLeft"/>
         </item>
         <item>
          <widget class="DapGuiLabel" name="DapGuiButtonTextMid_2">
           <property name="text">
            <string>Main Text</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="DapGuiLabel" name="DapGuiButtonTextRight_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Sub Text</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="DapGuiLabel" name="DapGuiButtonIconRight">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="DapGuiSeparator" name="kelGuiSeparator" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>4</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DapGuiSeparator</class>
   <extends>QWidget</extends>
   <header>dapguiseparator.h</header>
  </customwidget>
  <customwidget>
   <class>DapGuiLineEdit</class>
   <extends>QLineEdit</extends>
   <header>dapguilineedit.h</header>
  </customwidget>
  <customwidget>
   <class>DapGuiLineEditNative</class>
   <extends>QLineEdit</extends>
   <header>dapguilineeditnative.h</header>
  </customwidget>
  <customwidget>
   <class>DapGuiLabel</class>
   <extends>QLabel</extends>
   <header>dapguilabel.h</header>
  </customwidget>
  <customwidget>
   <class>DapGuiWidget</class>
   <extends>QWidget</extends>
   <header>dapguiwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DapGuiPushButton</class>
   <extends>QWidget</extends>
   <header>dapguipushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
